FogCreek/Glitch-Community
# Fix a couple popover styling bugs

## Links
* https://cookie-ricotta.glitch.me

## GIF/Screenshots:
before components: 
<img width="496" alt="Screen Shot 2019-10-30 at 4 04 59 PM" src="https://user-images.githubusercontent.com/3011231/67894559-2720b380-fb2f-11e9-9298-6955926ec79c.png">
current prod:
![Screen Shot 2019-10-30 at 4 05 20 PM](https://user-images.githubusercontent.com/3011231/67894575-2e47c180-fb2f-11e9-94b2-ab6f53659160.png)
fixed:
![Screen Shot 2019-10-30 at 4 05 31 PM](https://user-images.githubusercontent.com/3011231/67894593-36076600-fb2f-11e9-9771-00c28e1d1d21.png)

## Changes:
* Long titles weren't wrapping nicely, now they should be!
* There was also a bit of extra padding on the search results list in add-project-to-collection

## Additional Changes!
* removed some extra padding from the new-project-pop
* Fixed an error where the notification would claim you weren't able to create a collection with a project

## How To Test:
* Check out popovers related to long names (be it your username, or a project/collection)
* Create a new collection from a project page
